apps/api/src/deployment/routes/deployments/deployments.router.ts

```diff
@@ -78,6 +81,33 @@ const deleteRoute = createRoute({
   }
 });
 
+const depositRoute = createRoute({
+  method: "post",
+  path: "/v1/deployments/{dseq}/deposit",
```

Comment:
this endpoint is kinda relates to an entity in terms of REST. However this is rather a business flow endpoint. There is no need to complicate the path and better name the endpoint as a command. we discussed it [here](https://github.com/akash-network/console/discussions/888). E.g.

```suggestion
  path: "/v1/deposit-deployment",
```

and pass a dseq as a part of a body

Reply:
Makes sense